cli/pom.xml

```diff
@@ -77,11 +77,6 @@
       <artifactId>slf4j-jdk14</artifactId>
       <optional>true</optional> <!-- ditto -->
     </dependency>
-    <dependency> <!-- TODO remove and replace PrivateKeyProvider with SecurityUtils.createFileKeyPairProvider() as in SshClient -->
```

Comment:
FTR it was done by @kuisathaverat in https://github.com/jenkinsci/jenkins/pull/3902